CodeSniffer/Standards/Generic/Sniffs/PHP/SyntaxSniff.php

```diff
@@ -74,11 +74,11 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         }
 
         $fileName = $phpcsFile->getFilename();
-        $cmd      = $this->_phpPath." -l \"$fileName\" 2>&1";
+        $cmd      = $this->_phpPath." -l -d error_prepend_string='' \"$fileName\" 2>&1";
```

Comment:
The `error_append_string` won't cause any issues for this sniff as the regex does not use the `$` for end of line.